spec/spec_helper.rb

```diff
@@ -18,6 +18,8 @@
 SimpleCov.profiles.define 'integration' do
   add_filter 'spec/'
   add_filter 'cli.rb'
+  add_filter '../lib/oneview-sdk/image-streamer'
```

Comment:
I put this line because I implemented the i3s integration tests to run separately from the others. When tests for oneview resources run, simplecov would ignore archives related to the i3s.